lib/member_row.rb

```diff
@@ -7,7 +7,7 @@ class MemberRow < Scraped::HTML
   end
 
   field :photo do
-    URI.encode(noko[0].css('img/@src').text)
+    noko[0].css('img/@src').text.gsub(' ', '%20')
```

Comment:
Hmm, I think this is solving the right problem, but possibly at the wrong level!

If we added the `AbsoluteURLs` decorator to this class then that should handle the whitespace in URLs for you. However, the URLs are already absolute, which suggests that perhaps `AbsoluteURLs` has the wrong name, and should instead be called something like `CleanURLs`?

So I think the steps you need to take are:

1. Rename `AbsoluteURLs` to `CleanURLs` in scraped, probably best to open an issue for this as well.
2. Fix https://github.com/everypolitician/scraped/issues/62 
3. Update this scraper to use the `CleanURLs` decorator to silence the warnings